### PR TITLE
[13.0][MIG] purchase_stock_price_unit_sync: Migration to v13.0

### PR DESCRIPTION
**Pull request by carlosdauden**
@Tecnativa TT21654

### CODE REVIEW

**Comment by carlosdauden** [comment]
ping @sergio-teruel @chienandalu

**Review by sergio-teruel — APPROVED** [review]

File: purchase_stock_price_unit_sync/models/purchase_order.py [hunk 1]
```diff
@@ -17,9 +17,9 @@ def write(self, vals):
 
     def stock_price_unit_sync(self):
         for line in self.filtered(lambda l: l.state in ["purchase", "done"]):
-            line.move_ids.write(
+            line.move_ids.mapped("stock_valuation_layer_ids").write(
```
Comment (chienandalu) [inline purchase_stock_price_unit_sync/models/purchase_order.py:20]: Is it correct to write al these records at once?

Comment (sergio-teruel) [reply purchase_stock_price_unit_sync/models/purchase_order.py:20]: Yes!! in base module product_avco_cost_sync is recomputed all items..
ping @carlosdauden

**Review by chienandalu — APPROVED** [review]
ok :+1:

**Pull request merged**